src/includes/control.h

```diff
@@ -15,6 +15,7 @@
 #include <control-cmds.h>
 #include <service-listener.h>
 #include <cpbuffer.h>
+#include "control-datatypes.h"
```

Comment:
Hi, I reviewd your work and it looks good, unless that `control-datatypes.h` should included like this, I guess (with `< >` not `" "`):
```c++
#include <control-datatypes.h>
```

Reply:
Hey, nice catch thanks. Functionally it should not make a difference in this case. But I've changed the includes of control-datatypes.h so they are consistent with the others in the corresponding file.